proto/store.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "grpc.store";
+option csharp_namespace = "Momento.Protos.Store";
+
+package store;
+
+service Store {
+  rpc Get(_GetRequest) returns (_GetResponse) {}
+  rpc Set(_SetRequest) returns (_SetResponse) {}
+  rpc Delete(_DeleteRequest) returns (_DeleteResponse) {}
+}
+
+message _ValueType {
+  oneof type {
+    bytes bytes_value = 1;
+    string string_value = 2;
+    int64 integer_value = 3;
+    double double_value = 4;
+  }
+}
```

Comment:
I would call this message just `Value`
```
message _Value {
  oneof value {
    bytes bytes_value = 1;
    string string_value = 2;
    int64 integer_value = 3;
    double double_value = 4;
  }
}
```

Or may be we can directly use oneof in the request and response like this:
```
message _SetRequest {
  bytes key = 1;
  oneof value {
    bytes bytes_value = 2;
    string string_value = 3;
    int64 integer_value = 4;
    double double_value = 5;
  }
}

message _GetResponse {
  oneof value {
    bytes bytes_value = 1;
    string string_value = 2;
    int64 integer_value = 3;
    double double_value = 4;
  }
}
```

Reply:
Initially I was separating like this, but then I thought, maybe it makes more sense to define them outside since we don't want these two to have chance to diverge. What do you think?